stdlib/ballerina-builtin/src/main/ballerina/runtime/Package.md

```diff
@@ -0,0 +1,100 @@
+## Package overview
+
+This package includes functions to interact with the runtime, the invocation context and to manage errors.
+
+### Invocation Context
+
+The Invocation Context is a data holder which is created per request, and preserved for a single request-response flow. 
+The Invocation Context comprises of a unique id, UserPrincipal instance which includes user details and an AuthContext 
+instance which has the authentication related details if available.
+
```

Comment:
**Suggestion**
The Invocation Context is a data holder that is created per request and preserved for a single request-response flow.     
The Invocation Context comprises of a unique ID, a 'UserPrincipal' instance that includes user details and an 'AuthContext' instance that has the authentication related details if available.